mayavi/tests/common.py

```diff
@@ -21,6 +30,4 @@ def get_example_data(fname):
     """Given a relative path to data inside the examples directory,
     obtains the full path to the file.
     """
-    p = os.path.join('data', fname)
-    return os.path.abspath(fixpath(p))
-
+    return os.fspath(files("mayavi.tests") / "data" / fname)
```

Comment:
The motivating force behind this change is the failure seen here - https://github.com/enthought/mayavi/runs/3776106273?check_suite_focus=true

```
   ...
   File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\mayavi\core\file_data_source.py", line 199, in __set_pure_state__
    raise IOError(msg)
OSError: Could not find file at D:\C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\mayavi\tests\data\cube.vti
```

Note the `"D:\"` at the beginning of the file path which could not be found - but the package/environment is in `"C:\"`.

Reply:
This has temporarily been addressed but I've opened #1081 to track this issue